openpaas-james/deployment-tests/common/src/main/java/com/linagora/openpaas/deployment/CliContract.java

```diff
@@ -12,6 +12,8 @@
 
     @Test
     default void cliShouldWork() throws Exception {
+        Thread.sleep(2000);
```

Comment:
While being acceptable as a (very) short term solution, I would suggest putting an INFO log in `JMXServer.java` and awaiting that log here.

Can you contribute the log addition upstream in apache/james-project , and work asynchronously on removing this thread.sleep?